VoodooGPIO/VoodooGPIO.cpp

```diff
@@ -9,28 +9,24 @@
 
 #include "VoodooGPIO.hpp"
 
-OSDefineMetaClassAndStructors(VoodooGPIO, IOService);
+OSDefineMetaClassAndAbstractStructors(VoodooGPIO, IOService);
 
 IOWorkLoop *VoodooGPIO::getWorkLoop() {
     // Do we have a work loop already?, if so return it NOW.
-    if ((vm_address_t) workLoop >> 1)
-        return workLoop;
+    if ((vm_address_t) this->workLoop >> 1)
+        return this->workLoop;
```

Comment:
No need for `this`, the compiler will know that's part of the class.

Reply:
I added it for clarity, I find it a good practice. I can revert it